docs/sphinx/requirements.txt

```diff
@@ -1,8 +1,15 @@
 # cf. https://github.com/ryanfox/sphinx-markdown-tables/issues/36
 markdown<3.4.0
-sphinx==3.0.3
+sphinx==4.0.3
+katex
 recommonmark==0.6.0
--e git+git://github.com/pytorch/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme
 sphinxcontrib.katex
 torch
-sphinx-markdown-tables
+sphinx-markdown-tables==0.0.15
+torchserve
+torch-model-archiver
+torch-workflow-archiver
+
+# Also make sure to manually install - pytorch sphinx does not have a pypi repo
+# git clone https://github.com/pytorch/pytorch_sphinx_theme
+# python3 -m pip install -e pytorch_sphinx_theme
```

Comment:
I think to get the latest theme, you need to do:

https://github.com/pytorch/tutorials/blob/master/requirements.txt#L27

Reply:
This was hanging for me when I tried it out. Did it work for you?

Reply:
it seems to be working fine in the tutorials repo